modular_robot/revolve2/modular_robot/body/base/_active_hinge.py

```diff
@@ -0,0 +1,232 @@
+from __future__ import annotations
+
+from pyrr import Vector3
+
+from .._color import Color
+from .._module import Module
+from .._right_angles import RightAngles
+
+
+class ActiveHinge(Module):
+    """An Active Hinge."""
+
+    ATTACHMENT = 0
```

Comment:
```suggestion
    ATTACHMENT = 0

```

Seperate by enter for clarity